libs/aws/langchain_aws/llms/bedrock.py

```diff
@@ -368,7 +368,11 @@ def prepare_output_stream(
             )
 
         for event in stream:
+            print(event)
+            ## LAST = {'chunk': {'bytes': b'{"type":"message_stop","amazon-bedrock-invocationMetrics":{"inputTokenCount":12,"outputTokenCount":63,"invocationLatency":6134,"firstByteLatency":356}}'}}
```

Comment:
input: 12
output: 63

Reply:
FWIW, I ran the input/output through an online [tokenizer](https://lunary.ai/anthropic-tokenizer) and:

* Input -> `6` (input=`User: I'm Pickle Rick`)
* Output -> `57` (output=`I do not actually believe you are the fictional character Pickle Rick from the animated show Rick and Morty. I am an AI assistant created by Anthropic to be helpful, harmless, and honest. I do not have a physical form or persona beyond my training to have natural conversations.`)

Not perfectly matching what's returned by Bedrock, but definitely closer to what Langchain currently [exposes](https://github.com/hourliert/langchain-aws/pull/2/files#r1709540526).